src/main/java/com/minecolonies/entity/ai/citizen/miner/EntityAIStructureMiner.java

```diff
@@ -392,14 +396,14 @@ private BlockPos getNextBlockInShaftToMine()
         BlockPos nextBlockToMine = null;
         double   bestDistance    = Double.MAX_VALUE;
 
-        int xOffset = 3 * getOwnBuilding().getVectorX();
-        int zOffset = 3 * getOwnBuilding().getVectorZ();
+        int xOffset = SHAFT_RADIUS * getOwnBuilding().getVectorX();
+        int zOffset = SHAFT_RADIUS * getOwnBuilding().getVectorZ();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'zOffset' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)